### DISCUSSION TITLE
Not able to build using command

### DISCUSSION THREAD

**Discussion by abhiv01** [iTwin/itwinjs-core#4144] [General]
Hi,

Due to requirement of custom webpack config, we have run the command "nom run eject", and configuration added successfully, 

we are running Itwin application on webpack version **"4.44.2**",

but when I am trying to build using "npm run build", it is creating a problem,

and error coming as shown below: 

Module not found: Error: Can't resolve 'os' in '/Users/abhishek/Documents/officeWork/projects/digital-twin/node_modules/@itwin/core-common/lib/esm/rpc/web'
BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "os": require.resolve("os-browserify/browser") }'
        - install 'os-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "os": false }

Failed to parse source map from '/Users/abhishek/Documents/officeWork/projects/digital-twin/node_modules/@itwin/error-handling-react/src/components/fallback/FallbackUi.tsx' file: Error: ENOENT: no such file or directory, open '/Users/abhishek/Documents/officeWork/projects/digital-twin/node_modules/@itwin/error-handling-react/src/components/fallback/FallbackUi.tsx'

**Comment by pmconne** [comment]
The error text is telling you the problem and potential solutions. Did you try either of the suggested solutions?
The use of Node APIs like `os` in core-common will be addressed in iTwin.js 3.4.

**Comment by aruniverse** [comment]
You say you're using webpack version 4.44.2, but the error is with respect to webpack5. Can you confirm what version of webpack you're using? What did you eject from?
